alloydb-jdbc-connector/src/main/java/com/google/cloud/alloydb/ConnectionSocket.java

```diff
@@ -86,9 +86,18 @@ Socket connect() throws IOException {
             connectionInfo.getCertificateChain(),
             this.clientConnectorKeyPair.getPrivate());
 
-    String ipAddress = connectionInfo.getIpAddress();
-    if (connectionConfig.getIpType().equals(IpType.PUBLIC)) {
-      ipAddress = connectionInfo.getPublicIpAddress();
+    String ipAddress;
```

Comment:
Could we change this to address? When it's public or private, it will be an IP. But PSC will always be DNS.

Reply:
updated.